etc/docker/dev/README.rst

```diff
@@ -4,83 +4,117 @@ Setting up a Rucio development environment
 
-Container for Development
+You can confirm that Docker is running properly by executing (might need `sudo`)::
+
+    docker run hello-world
+
+If successful, this will print an informational message telling you that you are ready to go.  Now, also install the `docker-compose` helper tool, e.g., with `sudo apt install docker-compose`, and then start the Docker daemon with `sudo systemctl start docker`. You are now ready to install the Rucio development environment.
+
+Preparing the environment
 -------------------------
 
-The Dockerfile in this container is used to centrally produce the docker container available as rucio/rucio-dev. It
-provides a rucio environment which allow you to mount your local code in the containers bin, lib, and tools directory. The
-container is set up to run against a mysql database called rucio with user rucio, password rucio. Tests and checks may be
-run against the development code without having to rebuild the environment for each test.
+This container can be found on Dockerhub as `rucio/rucio-dev`, and the corresponding `Dockerfile <https://https://github.com/mlassnig/containers/tree/master/dev>`_ is also available. It provides a Rucio environment which allows you to mount your local code in the containers `bin`, `lib`, and `tools` directory. The container is set up to run against a PostgreSQL database. Tests and checks can be run against the development code without having to rebuild the container.
 
-docker-compose
---------------
+The first step is to fork the `main Rucio repository on GitHub <https://github.com/rucio/rucio>`_ by clicking the yellow Fork Star button, and then clone your private forked Rucio repository to your `~/dev/rucio`. Afterwards add the main upstream repository as an additional remote to be able to submit pull requests later::
 
-YAML for docker compose has been provided to allow easy setup the containers from the rucio code directory::
+    cd ~/dev
+    git clone git@github.com:<your_username>/rucio.git
+    cd rucio
+    git remote add upstream git@github.com:rucio/rucio.git
+    git fetch --all
 
-   $> docker-compose --file etc/docker/dev/docker-compose.yml up -d
+Now, ensure that the `.git/config` is proper, i.e., mentioning your full name and email address, and that the `.githubtoken` is correctly set. Optionally, you can also replace the `~/dev/rucio/tools/pre-commit` hook with the one provided `here <https://github.com/rucio/rucio/blob/master/etc/docker/dev/pre-commit>`_ so that `pylint` run in the container rather then in the local system.
```

Comment:
you are deleting `etc/docker/dev/pre-commit` with this PR... add it to the containers repo and point to that one.